css/mainPage.css

```diff
@@ -0,0 +1,58 @@
+.container {
+    background-color: antiquewhite;
+    margin: auto;
+}
+
+.logoContainer {
+    min-height: 200px;
+}
+
+.descriptionText {
+    min-height: 100px;
+    font-weight: 500;
+    font-size: 25px;
+    font-style: italic;
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+}
+
+.search {
+    margin-bottom: 100px;
+}
+
+.recentSearches {
+    min-height: 100px;
+}
+
+html, body {
+    height: 100%;
+    background-color: snow;
+}
+
+.container-fluid {
+    height: 100%;
+    display: flex;
+}
+
+.searchInput {
+    height: 100%;
+    background-color: rgb(206, 71, 71);
+    color: white;
+}
+
+.searchInput:focus {
+    background-color: rgb(206, 71, 71);
+    color: white;
+}
+
+.searchInput::placeholder {
+    color: white;
+}
+
+.searchBtn {
+    background-color: rgb(206, 71, 71);
+    border-color:  rgb(206, 71, 71);
+}
+
+.recentItem {
+    background-color: rgb(206, 71, 71);
```

Comment:
There are a few repeated colours so it would be best to put them in a variable to recall later.
Also, best practice is to standardize colour format to hex values (#000000) as they tend to process faster.

Reply:
we'll do this once we put the CSS together